.storybook/main.ts

```diff
@@ -0,0 +1,47 @@
+import { StorybookConfig } from '@storybook/angular';
+import remarkGfm from 'remark-gfm';
+
+const config: StorybookConfig = {
+  framework: {
+    name: '@storybook/angular',
+    options: {},
+  },
+  stories: [],
+  addons: [
+    '@storybook/addon-controls',
+    {
+      name: '@storybook/addon-storysource',
```

Comment:
Had to use `@storybook/addon-storysource` instead of `@whitespace/storybook-addon-html`, to get rid of ambiguous "TypeError: g.trim is not a function" error while opening the storybook.

Though it is not possible to see the source html in addons "Code" tab, because of an open issue that the `@storybook/addon-storysource` currently has : https://github.com/storybookjs/storybook/issues/22922

Reply:
No, it should be OK.